style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.menu-toggle {
+  display: none;
+}
+
+nav .container {
+  background-color: #3e3c3c;
+  display: flex;
+  height: 72px;
+}
+
+.container .name-info {
+  flex: 3;
+}
+
+.container ul {
+  display: flex;
+  justify-content: space-around;
+  flex: 1;
+}
+
+.list-container li {
+  list-style: none;
+  text-decoration: none;
+}
+
+@media screen and (max-width: 600px) {
+  main {
+    background-color: #1c1a19;
+    padding: 20px;
+    height: 100vh;
+  }
+
+  nav .container {
+    background-color: #3e3c3c;
+    display: flex;
+    height: 90px;
+    padding: 10px;
+    align-content: center;
+    align-items: center;
+  }
+
+  .bio-container h1 {
+    display: flex;
+    color: #ff6b00;
+    flex-wrap: wrap;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    order: 0;
+    flex-grow: 0;
+    margin: 12px 0;
+  }
+
+  .paragragh p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    margin: 12px 0;
+  }
+
+  .bio-container h3 {
+    color: #fff5e1;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    margin: 12px 0;
+  }
+
+  .container .name-info {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    color: #fff;
+    flex: 10;
+  }
+
+  .menu-toggle {
+    display: block;
+    flex: 1;
+  }
+
+  .container ul {
+    display: none;
+  }
+
+  .bio-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
```

Comment:
- [ ] Kindly add background to this section, to match the Figma design.
- [ ] Kindly add some padding-top to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/158718728-2b2e7c1e-41b6-4c55-8ce6-0ef361321c24.png) | ![image](https://user-images.githubusercontent.com/35535130/158718826-ac604c5e-7516-4395-abd1-8f2b2fec6d73.png) |

Reply:
Added the background color and also the padding top